src/Stackctl/StackSpecYaml.hs

```diff
@@ -76,10 +86,8 @@ instance ToJSON ParameterYaml where
   toEncoding = pairs . mconcat . parameterPairs
 
 parameterPairs :: KeyValue a => ParameterYaml -> [a]
-parameterPairs (ParameterYaml p) = fromMaybe [] $ do
-  k <- p ^. parameter_parameterKey
-  v <- p ^. parameter_parameterValue
-  pure ["ParameterKey" .= k, "ParameterValue" .= v]
+parameterPairs (ParameterYaml k v) =
+  ["ParameterKey" .= k, "ParameterValue" .= v]
```

Comment:
We could emit `[ k .= v ]` here, if we want to use the natural syntax in Capture/Generate too.

I'd make `_pvKey :: Aeson.Key` too.